### PR TITLE
Package create test

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: src/api/spec/features/webui/projects_spec.rb [hunk 1]
```diff
@@ -35,13 +35,74 @@
...
+      fill_in "name", :with => "coolstuff"
+      click_button "Save changes"
+
+      expect(page).to have_text("You can't create packages in #{user.home_project_name}")
+      expect(page.current_path).to eq("/project/new_package/#{user.home_project_name}")
+    end
+
+    scenario "as admin" do
+      login admin_user
+      visit project_show_path(project: user.home_project_name)
```
Comment (hennevogel) [inline src/api/spec/features/webui/projects_spec.rb]: This is not really testing what you say you test right? Please really use a project in the *global namespace*

**Comment by hennevogel** [comment]
Are you sure you need all the different requests in the cassettes?

File: src/api/spec/features/webui/projects_spec.rb (hunk 1 above)
Comment (bgeuken) [inline src/api/spec/features/webui/projects_spec.rb]: global namespace was just an example. But I can change that.

**Comment by bgeuken** [comment]
Let me check

**Comment by bgeuken** [comment]
You were right. I cleaned up the cassetes now and updated the permission test

File: src/api/spec/features/webui/projects_spec.rb [hunk 2]
```diff
@@ -35,13 +35,75 @@
...
+    scenario "with invalid data (validation fails)" do
+      fill_in "name", :with => "cool stuff"
+      click_button "Save changes"
+
+      expect(page).to have_text("Invalid package name: 'cool stuff'")
+      expect(page.current_path).to eq("/project/new_package/#{user.home_project_name}")
+    end
+
+    scenario "that already exists" do
+      create(:package, name: "coolstuff", project: Project.find_by_name(user.home_project_name))
```
Comment (mdeniz) [inline src/api/spec/features/webui/projects_spec.rb]: use `home_project` here like in the `before` section:
```ruby
 create(:package, name: "coolstuff", project: user.home_project)
```

**Comment by mdeniz** [comment]
Beside the nitpick it LGTM :+1:

**Pull request merged**